ui/src/app/utils/runtime-utils.tsx

```diff
@@ -1291,4 +1291,5 @@ export const isActionable = (status: RuntimeStatus) =>
   ).includes(status);
 
 export const getCreator = (runtime: ListRuntimeResponse): string | undefined =>
-  (runtime?.labels as Map<string, string>).get('creator');
+  // eslint-disable-next-line @typescript-eslint/dot-notation
```

Comment:
Leonardo returns this as a plain `Object` so we have no type info about whether the creator field exists.  I have a test case for when the field doesn't exist - this returns undefined instead of erroring.